create-turbo/src/shouldUsePnpm.ts

```diff
@@ -1,3 +1,4 @@
+import semver from "semver";
```

Comment:
Do we need the full dependency here? Can we just strip the first character, convert to number and see if it is less than?

Reply:
solved